context/app/static/js/components/searchPage/SearchWrapper.jsx

```diff
@@ -52,11 +52,7 @@ function SearchWrapper(props) {
     const removalFn = searchkit.addResultsListener((results) => {
       const newQueryString = searchkit.state?.q;
       if (![queryString.current, undefined].includes(newQueryString)) {
-        trackEvent({
-          category: analyticsCategory,
-          action: 'Free Text Search',
-          label: newQueryString,
-        });
+        trackSiteSearch(newQueryString);
```

Comment:
Not entirely sure what `trackSiteSearch` handles for us, but how can we differentiate 'site searches' as we have in the past with Google Analytics' category and action?

Reply:
`trackSiteSearch` has an optional `category` param, which I'm now providing.